minio/policy.py

```diff
@@ -132,7 +132,7 @@ def _get_in_use_policy(statements, bucket_name, prefix=''):
         resource = s['Resource']
         actions = set(_get_action(s))
         if (resource != object_resource and
-                resource.startswith(resource_prefix)):
+                resource[0].startswith(resource_prefix)):
```

Comment:
1. As per Amazon AWS dynamic JSON, `Resource` can be a string or a list of string.  Assuming as a list would break logically.  The right fix would be type checking like
```py
resource = s['resource']
if isinstance(resource, basestring):
    resource = [resource]
```

2. We shouldn't assume `resource[0]` instead it could match in any item in the list.